flocx_market/api/bid.py

```diff
@@ -1,47 +1,36 @@
 from flask_restful import Resource
 from flask import request
-from flocx_market.db.sqlalchemy import api as dbapi
-
+import flocx_market.objects.bid as bo
 
 class Bid(Resource):
 
     @classmethod
     def get(cls, marketplace_bid_id=None):
-        if marketplace_bid_id is None:
-            return BidList.get()['bids']
+        a = bo.get_bid(marketplace_bid_id)
```

Comment:
If you update the code to the comments I have on objects/bid.py, this becomes `bid.Bid.get(marketplace_bid_id`